docs/index.md

```diff
@@ -4,12 +4,15 @@ layout: ""
 page_title: "Provider: FMC"
 description: |-
   The FMC provider provides resources to interact with a Cisco Secure FMC (Firewall Management Center) instance.
+
 ---
 
 # FMC Provider
 
 The FMC provider provides resources to interact with a Cisco Secure FMC (Firewall Management Center) instance. It communicates with FMC via the REST API.
 
+We strongly recommend to use `-parallelism=1` option with Terraform. The FMC REST API currently does not support parallel create/update/delete operations.
```

Comment:
If the API does not support concurrent write operations, I would prefer to enforce that with a semaphore at the client level, instead of forcing people to use the parallelism option in TF. Which we don't have to do in this PR but if you agree I would suggest to remove the comment for now.

Reply:
Agreed, I think a client-level mutex is the way to go. I'll get back to you about removing this remark from the docs.